src/NuGetGallery.Core/GitHub/NuGetPackageGitHubInformation.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace NuGetGallery.GitHub
+{
+    public class NuGetPackageGitHubInformation
+    {
+        public readonly static NuGetPackageGitHubInformation Empty = new NuGetPackageGitHubInformation(
+                                                            0,
+                                                            new List<RepositoryInformation>());
+
+        public NuGetPackageGitHubInformation(int totalRepos, IReadOnlyList<RepositoryInformation> repos)
+        {
+            if (totalRepos < 0)
+            {
+                throw new IndexOutOfRangeException(string.Format("{0} cannot have a negative value!", nameof(totalRepos)));
+            }
+
+            TotalRepos = totalRepos;
+            Repos = repos ?? throw new ArgumentNullException(nameof(repos));
+        }
+
+        public int TotalRepos { get; }
```

Comment:
It's unclear what these fields contain without looking at `GitHubUsageConfiguration`. Two options:

1. You can add comments and clarify
2. You can make this class a little more restrictive. If so, I would move `ReposPerPackage` to this class